.travis.yml

```diff
@@ -12,10 +12,12 @@ env:
   - TRAVIS_GOOS=windows
   - TRAVIS_GOOS=linux
   - TRAVIS_GOOS=darwin
+  - TRAVIS_GOOS=solaris
 
 script:
 # TODO: vendor
   - export GOOS=${TRAVIS_GOOS}
   - go get -t ./...
   - make build binaries
   - if [ "$GOOS" = "linux" ]; then make vet test; fi
+  - if [ "$GOOS" = "solaris" ]; then go test -c; fi
```

Comment:
Create a target `make compile` test so the flags and env get setup consistently.